source/sync/learn/overview.txt

```diff
@@ -380,6 +380,13 @@ Production Load Testing
   Measure performance and identify issues in a scaled-up production
   deployment with :ref:`sync-production-load-testing`.
 
+Write Transaction Size
+  When writing lots of data, consider using multiple small write
```

Comment:
"lots of" or "a lot of" or "a large amount of"? The first seems a bit too informal to me.